src/lib.rs

```diff
@@ -412,7 +412,7 @@ impl Context {
 
         if ret.is_success() {
             let signature = unsafe { MBox::from_raw(signature) };
-            Ok((*signature).try_into()?)
+            Ok(unsafe { Signature::try_from(*signature)? })
```

Comment:
Maybe a comment to explain why this is safe would make us better `unsafe` citizens 👷

Reply:
Same everywhere we use the `unsafe` block actually in my opinion. The same way we kind of have to justify when we panic explicitely.

Reply:
Indeed, will add